hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -1070,9 +1070,7 @@
 
             All other values are reserved.
 
-            The SW may advertises as supporting more than `max_wr_size` to the South Brige.
-            If the SW reports the transfer size more than the value described in this field,
-            the SW must read the data from the write FIFO before the FIFO becomes full in order not to make FIFO overflow.
+            It is not recommended for SW to advertise TPM supporting more than `max_wr_size` to the South Bridge.
```

Comment:
Oh, I just noticed: Was this blurb supposed to go in the "max_rd_size" section? It looks like these bits are for "max_wr_size"

Reply:
I do not know where I addressed this. But should be in somewhere in other PRs. Hum... I could not find it .. Maybe it would be disappeared while I was rebasing.. let me quick-fix it

Reply:
shoot. I didn't upload the commit :( Let me amend here.

Reply:
Thanks, @eunchan!